Decred-Never-Stops.md

```diff
@@ -0,0 +1,100 @@
+Politeia: Exploring the role of Politeia, Decred's proposal system, in generating a sustainable work environment by providing a platform for transparent discussion and funding allocation.
+
+Continuous Improvement:
+
+Talk about Decred github repository and Decred applications
+
+Marketing and Outreach Efforts:
+
+Building awareness: Exploring Decred's marketing and outreach efforts to raise awareness and attract new users and investors.
+Engaging the community: Highlighting Decred's initiatives to engage and educate its community through events, social media, and educational resources.
+
+Overcoming Challenges:
+
+Addressing obstacles: Identifying common challenges faced by Decred and how the project proactively addresses them through strategic planning and resource allocation.
+
+Looking to the Future:
+
+Vision for growth: Sharing Decred's vision for the future and its plans for continued development and expansion.
+
+# Article / TitleDecred's Eternal Drive: Perseverance in the Face of Crypto Challenges
+
+Text: 
+In the unpredictable world of cryptocurrency, where projects often rise and fall with the tides of market sentiment, one project stands out for its commitment to progress: Decred. Born from a proposal on the Bitcointalk forum back in April 2013, Decred emerged as a beacon of innovation and resilience in the crypto space.
+
+At its core, Decred was envisioned not just as another digital currency, but as a new project/workplace for the future of decentralized finance. Spearheaded by visionary minds like tacotime, _ingsoc, and Jake Yocom-Piatt, Decred took shape as a hybrid of Proof-of-Work (PoW) and Proof-of-Stake (PoS) consensus systems, drawing inspiration from the Memcoin2 cryptocurrency.
+
+Decred's journey isn't just about creating a new form of currency; it is about building an ecosystem where innovation thrives. Decred's hybrid model not only ensures security and decentralization but also fosters a culture of continuous development and adaptation. Let's check what makes Decred a unique project!
+
+A Legacy of Innovation
+
+The innovation lies at the core of the Decred project, driving its pursuit of progress in the blockchain ecosystem. Formed by a team of exceptional developers, where many have worked for big companies and coded for big projects, such as Bitcoin, Decred stands out as a fertile ground for forward-thinking solutions and blockchain applications. Decred's unique hybrid consensus model is central to its ability to stay at the forefront of innovation which enables swift adoption of new technologies and methodologies.
+
+Decred's network operates on a unique hybrid consensus protocol, blending two key approaches: Proof of Work (PoW) and Proof of Stake (PoS). In this setup, PoW miners handle the validation of transactions, while PoS stakeholders take charge of voting on network upgrades. This innovative model ensures that PoW miners and PoS stakeholders are rewarded, with block rewards being divided between them.
```

Comment:
"PoW miners handle the validation of transactions, while PoS stakeholders take charge of voting on network upgrades." This isn't an accurate description of how the process works. For instance, here's a basic outline of the process:

1. Full nodes build a list of transactions to go into future blocks. 
2. Full nodes check that each transaction fits the requirements / rules e.g. paid their fee, come from an address that actually has the funds available and a basic check to make sure there is no double spending.
3. The miner tries to solve a puzzle. If they solve the puzzle they get to build the next block
4. The miner then takes the transaction list that's been created by the network full nodes and stuff as many transactions as possible into the block that they are allowed to build.
5. This block then enters into the Decred blockchain as un-mined and awaiting verification
6. Upto five randomly picked POS tickets then validate the block against the consensus rules. If the block passes these checks it's enter into the blockchain.

This is a basic overview of the process but as you can see the work done by POS is a secondary validator that checks the miners work. Having this double check prevents bad mining behaviour like building empty blocks, filling blocks with transactions that go against the rules of the network etc..